src/api/modules/utils.ts

```diff
@@ -129,81 +128,99 @@ export const appIdFromRegistry = (app: string, majorLocator: string) => {
     .catch(handleError(app))
 }
 
-export function optionsFormatter(billingOptions: BillingOptions) {
+const FREE_BILLING_OPTIONS_TYPE = 'free'
+export const isFreeApp = (type?: string, free?: boolean) => type === FREE_BILLING_OPTIONS_TYPE || free
```

Comment:
You could also receive the billing options directly
```suggestion
export const isFreeApp = ({ type, free }: BillingOptions) => type === FREE_BILLING_OPTIONS_TYPE || free
```

Reply:
I did that way befefore, although the code looks nicer I think it makes maintainability worse IMHO. If you want to search for all uses of "billingOptions" you would find that line, but that function doesn't use the whole billingOptions

Reply:
I see. I just thought It could be used as something like "give me your billing options and I'll find out if this is a free app". One middle ground I know is do this:
```suggestion
export const isFreeApp = ({ type, free }: { type?: string, free?: boolean }) => type === FREE_BILLING_OPTIONS_TYPE || free
```
Then, you can call this by providing the billing options, while also being specific with what you really care about

Reply:
Nice, thats better! Done in 25283acc92ee221ef8640460eb39d837c48b5692 , thanks!